app/util/fetchUtils.js

```diff
@@ -34,4 +34,19 @@ export const retryFetch = (URL, options = {}, retryCount, retryDelay) =>
     retry(retryCount);
   });
 
-export default retryFetch;
+export const fetchWithHeaders = (URL, headers) => {
+  return fetch(URL, {
+    headers: { ...headers },
+  }).then(res => {
+    if (res.status !== 200) {
+      return undefined;
+    }
+    return res;
+  });
+};
+
+export const fetchWithLanguage = (URL, language, addHeaders = {}) => {
+  const headers = { 'Accept-Language': language, ...addHeaders };
+  return fetchWithHeaders(URL, headers);
+};
+export default { retryFetch, fetchWithHeaders };
```

Comment:
Default export this should be removed as module fetchUtil doesn't sound like it provides any natural export primitive and there's no default import statement to this in the codebase - yet.